css/easyddb_legend.css

```diff
@@ -16,19 +14,26 @@
 }
 
 .availability-legend-item img {
-  height: 10px;
-  width: 10px;
+  height: 15px;
+  width: 15px;
   margin-right: 0.25em;
+  margin-top: -1px;
+  float: left;
+  border-radius: 5px;
 }
-
+.availability.available,
+.availability.available.reservable,
 .availability-legend-item.available img {
   background-color: #528c10; /* Green */
 }
 
+.availability.reservable,
```

Comment:
What about:

- `.availability.processed,`
- `.availability.available.not-reservable,`
- `.availability.unavailable.reservable,`

?